src/components/banner/banner.scss

```diff
@@ -267,53 +340,29 @@
     }
   }
 
-  .headline {
-    z-index: 1;
-
-    a {
-      &:hover {
-        text-decoration: none;
-      }
-
-      &:focus {
-        text-decoration: underline;
-      }
-    }
-
-    &.headline--negative {
-      color: $secondary;
-
-      a {
-        color: #fff;
-      }
-
-      text-shadow: 0 1px 1px #000;
-      color: #fff;
-    }
-    margin-top: 0;
-    margin-bottom: .7rem;
-  }
 }
 
 // Set large heading size
-.headline--large.headline--serif,
-.headline--large {
-  font-size: 4.7rem;
-  font-size: containerClamp(2.5rem, 4.7rem);
-}
+:where(h2, h3, h4, h5, h6) {
+  &.headline--large.headline--serif,
+  &.headline--large {
```

Comment:
I'm wondering why both of these lines are needed?

Reply:
Good question, it is to override the modification to the sizes set in https://github.com/uiowa/uids/blob/main/src/components/typography/headings/headings.scss to make Zilla render as a comparable size to Roboto.

Reply:
I removed one set that wasn't needed and added some supporting comments.  Thanks for pointing it out!